rigelprotocol/SmartswapDappV2
# Link manage pool and add liquidity buttons

#### What does this PR do?
- This PR links manage pool and add liquidity buttons to the respective pages

#### What has been completed?
- [x] Links manage pool and add liquidity button 
- [x] Corrects spelling mistake on the ErrorBoundary folder name

#### How should the changes be manually tested?
- Click on the deployed PR link and navigate to the find pool page, 
- Click select tokens to find pool 
- For pool that exist, the manage pool button is displayed
- which can be clicked and it navigates to the pool page for the user to manage pool

#### Any Background context or information you want to provide?
- N/A

#### Demo Video and screenshots?
- https://www.loom.com/share/475c2d7d849e46d68bee5bee4adbf787
